values/jobs/harbor.gotmpl

```diff
@@ -15,9 +15,10 @@
 {{- $teamNames = print "team-" . | append $teamNames -}}
 {{- end -}}
 
-tasks:
+jobs:
   harbor:
```

Comment:
now that we don't allow external overrides of core jobs anymore (we never should have in the first place), we can make this an array of jobs, as opposed to a hash with the name as key. That way we conform to the rest of the work